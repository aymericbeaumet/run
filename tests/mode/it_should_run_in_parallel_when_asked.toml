mode = "parallel"

[[run]]
name = "finish second"
cmd = ["timeout", "-v", "0.2s", "cat"]

[[run]]
name = "finish first"
cmd = ["timeout", "-v", "0.1s", "cat"]
