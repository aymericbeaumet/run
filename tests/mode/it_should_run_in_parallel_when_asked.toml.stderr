[finish first] timeout: sending signal TERM to command ‘cat’
[finish second] timeout: sending signal TERM to command ‘cat’
